cephfs/admin/fs_quiesce_test.go

```diff
@@ -0,0 +1,50 @@
+//go:build ceph_preview && squid
+
+package admin
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFSQuiesce(t *testing.T) {
+	fsa := getFSAdmin(t)
+	volume := "cephfs"
+	group := NoGroup
```

Comment:
define subvolume name in a variable as we are using `"quiesceMe"` in multiple places?